### PR TITLE
Spring-boot basics - Introducing skeleton for spring data, example repository, temporary controller

### PR DESCRIPTION
**Pull request by wmilczarek**

### CODE REVIEW

**Comment by ThePnx** [comment]
+537 ? Please split it to 10 commits, max 54 lines per one.

**Comment by ThePnx** [comment]
gradle-wrapper.jar should be excluded in git ignore.

**Comment by ThePnx** [comment]
:+1: if you exclude gradle-wrapper.jar

**Review by ThePnx — CHANGES_REQUESTED** [review]
:+1:

File: src/main/java/rotate/it/data/model/PullRequest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    private Team team;
+
+    @Column(nullable = false)
+    private String creationTimestamp;
+
+    public String getPullRequestId() {
+        return pullRequestId;
+    }
+
+    public void setPullRequestId(String pullRequestId) {
```
Comment (ThePnx) [inline src/main/java/rotate/it/data/model/PullRequest.java:38]: Do we really need setters?

**Review by ThePnx — COMMENTED** [review]
:+1:

**Review by ThePnx — APPROVED** [review]

**Comment by wmilczarek** [comment]
I have no idea what we will need in near future and you?

**Pull request merged**